<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>无符号大整数计算器</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFirst">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>第一个数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSecond">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>第二个数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFirst">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSecond">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>结果：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxOperations">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>681</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QGroupBox" name="groupBoxBinaryOperations">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>311</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>数值运算</string>
     </property>
     <widget class="QPushButton" name="pushButtonPlus">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMinus">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMultiply">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDivision">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonModulo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxRelationalOperations">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>311</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>关系运算</string>
     </property>
     <widget class="QPushButton" name="pushButtonGreater">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLess">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonGreaterEqual">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>≥</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLessEqual">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>≤</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEqual">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonNotEqual">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>≠</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowserResult">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
